### PR TITLE
Maj ClickOnce packages path location for VS 2017

### PR DESCRIPTION
**Pull request by naster01**
The path has been changed for VS 2017.

### CODE REVIEW

File: docs/deployment/how-to-include-prerequisites-with-a-clickonce-application.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ Before you can distribute prerequisite software with a [!INCLUDE[ndptecclick](..
 
 1. In File Explorer, open the **Packages** folder.
 
-    By default, the path is *C:\Program Files\Microsoft Visual Studio 14.0\SDK\Bootstrapper\Packages* on a 32-bit system and *C:\Program Files (x86)\Microsoft Visual Studio 14.0\SDK\Bootstrapper\Packages* on a 64-bit system.
+    By default, the path is : 
+      - For VS2015 : *C:\Program Files\Microsoft Visual Studio 14.0\SDK\Bootstrapper\Packages* on a 32-bit system and *C:\Program Files (x86)\Microsoft Visual Studio 14.0\SDK\Bootstrapper\Packages* on a 64-bit system.
```
Comment (gewarren) [inline docs/deployment/how-to-include-prerequisites-with-a-clickonce-application.md]:
```suggestion
```

File: docs/deployment/how-to-include-prerequisites-with-a-clickonce-application.md [hunk 2]
```diff
@@ -19,7 +19,9 @@ Before you can distribute prerequisite software with a [!INCLUDE[ndptecclick](..
 
 1. In File Explorer, open the **Packages** folder.
 
-    By default, the path is *C:\Program Files\Microsoft Visual Studio 14.0\SDK\Bootstrapper\Packages* on a 32-bit system and *C:\Program Files (x86)\Microsoft Visual Studio 14.0\SDK\Bootstrapper\Packages* on a 64-bit system.
+    By default, the path is :
```
Comment (gewarren) [inline docs/deployment/how-to-include-prerequisites-with-a-clickonce-application.md]:
```suggestion
    By default, the path is *%ProgramFiles(x86)%\Microsoft SDKs\ClickOnce Bootstrapper\Packages\*.
```

File: docs/deployment/how-to-include-prerequisites-with-a-clickonce-application.md [hunk 3]
```diff
@@ -19,7 +19,9 @@ Before you can distribute prerequisite software with a [!INCLUDE[ndptecclick](..
 
 1. In File Explorer, open the **Packages** folder.
 
-    By default, the path is *C:\Program Files\Microsoft Visual Studio 14.0\SDK\Bootstrapper\Packages* on a 32-bit system and *C:\Program Files (x86)\Microsoft Visual Studio 14.0\SDK\Bootstrapper\Packages* on a 64-bit system.
+    By default, the path is : 
+      - For VS2015 : *C:\Program Files\Microsoft Visual Studio 14.0\SDK\Bootstrapper\Packages* on a 32-bit system and *C:\Program Files (x86)\Microsoft Visual Studio 14.0\SDK\Bootstrapper\Packages* on a 64-bit system.
+      - For VS2017 : *C:\Program Files (x86)\Microsoft SDKs\ClickOnce Bootstrapper\Packages\*
```
Comment (gewarren) [inline docs/deployment/how-to-include-prerequisites-with-a-clickonce-application.md]:
```suggestion
```

**Comment by Mikejo5000** [comment]
#sign-off

**Comment by Mikejo5000** [comment]
@naster01 Thanks for the fix!

**Comment by PRMerger15** [comment]
@gewarren : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Jak-MS : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Jak-MS : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Jak-MS : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

**Comment by Jak-MS** [comment]
@naster01 @Mikejo5000 - the asterisks were displaying and not working as markdown. i used backticks for the path... it displays well now. if we need to change this, follow up and we'll get it changed.

**Pull request merged**